tgui/packages/tgui/interfaces/ghost_crew_manifest.js

```diff
@@ -0,0 +1,18 @@
+import { Fragment } from 'inferno';
+import { useBackend } from '../backend';
+import { Box, Section } from '../components';
+import { Window } from '../layouts';
+import { CrewManifest } from "./common/CrewManifest";
+
+export const ghost_crew_manifest = (props, context) => {
```

Comment:
The file name and class name should be in PascalCase to keep consistency (`GhostCrewManifest`)